src/dash-adapter.js

```diff
@@ -769,6 +778,7 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
             })
             .then(() => {
               const data = {tracks: this._getParsedTracks()};
+              this._maybeCapLevelToPlayerSize();
```

Comment:
so we not supporting change the capLevelToPlayerSize flag on the fly?

Reply:
Yes, we won't.
Also on hls we won't